app/views/companies/dashboard.html.erb

```diff
@@ -1,18 +1,20 @@
 <div class="row">
-  <div class="col-5">
+  <div class="col-lg-3">
     <%= render 'layouts/sidebar' %>    
   </div>
   <!-- User Settings Graphs and History Enable and Disable-->
-  <div class="col-7 p-3">
-    <% if @user_setting.show_graphs? %>
-      <div id="graphs">
-        <h1>Graphs</h1>
-      </div>
-    <% end %>
-    <% if @user_setting.show_history? %>
-      <div id="histroy">
-        <h1 >History</h1>
-      </div>
-    <% end %>
+  <div class="col-lg-9">
+  <% if @user_setting.show_graphs? %>
+    <div id="graphs" class="my-2">
+      <h1>Graphs</h1>
```

Comment:
move to en.yml (all hard coded messages)

Reply:
This is just a heading? Should we move it too?